src/common/options.cc

```diff
@@ -1477,6 +1477,11 @@ std::vector<Option> get_global_options() {
     .set_default(65536)
     .set_description(""),
 
+    Option("mon_max_pool_pg_num_per_osd", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    .set_default(100)
```

Comment:
this seems low - could make it 300 to match the warning on existing pools ```mon_pg_warn_max_per_osd```... maybe just use that option for this check instead of adding a new one